packages/api/src/interfaces/cli/Osiris.cli.ts

```diff
@@ -122,16 +122,18 @@ export default class OsirisCli {
         const requests = OsirisParser.parseRequests(contentFile, year);
 
         let tictackClock = true;
-        const results = await requests.reduce(async (acc, osirisRequest, index) => {
-            const data = await acc;
-
+        const ticTacInterval = setInterval(() => {
```

Comment:
Hum, je me demande si ça pas été mis dans la boucle au cas ou sa planterai dans la boucle ?

Reply:
Ce qu'il me semble que tu m'avais dit c'était que ça permettait d'éviter que scalingo croie qu'il ne se passe plus rien parce que l'affichage n'évoluerait pas. Je vois pas l'intérêt du tic tac pour le cas où ça planterait

Reply:
Si une erreur no catch arriverai ça bloquerai le script et donc plus de tic tac qui permettrais a scalingo de kill l'app et pour le coup ils aurait raisons, non ?

Reply:
Si une erreur arrive ça interrompt le script donc on a l'occasion de s'en rendre compte et ça compte aussi mes tic tac. Pas besoin d'avoir un tic/tac par itération pour ça..?

Reply:
Ok, j'ai l'impression qu'on zape un cas mais, je vois plus le quelle